docs/main.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable */ (() => {
```

Comment:
-  [ ] OPTIONAL: It is very common to name the distribution folder `dist` instead of `docs` for better terminology. If you are having with deploying to github_pages, it is not a requirement in this milestone but if you need it, you can follow this [link](https://gist.github.com/cobyism/4730490) to learn how to deploy a sub-tree in a project (`dist`) alone without any problem.

Reply:
Thanks a lot, The deployment gave me a lot of headache.